### DISCUSSION TITLE
Thanks for continuing to maintain this tool!

### DISCUSSION THREAD

**Discussion by WilfulTravesty** [RamonUnch/AltDrag#554] [General]
Just wanted to express my thanks for continuing to maintain this useful tool!

**Comment by nilslindemann** [comment]
I second this. AltDrag was and AltSnap is the only tool which properly goes full screen and then restores to the snapped position when clicking the middle of the window buttons. This is so obvious and useful, but for reasons I do not know, no other window manager with snap functionality, including those of Linux, does it like this. I can not return to the snapped state using this button, which is annoying, as I need this functionality dozens of times daily, when having two windows side by side and making one full screen for a moment, then restoring it to its snapped state. AltSnap makes this possible, so it is nice that it lives!

**Comment by dillacorn** [comment]
I'm personally excited for 1.64 update because then this application should work natively with GlazeWM and for the first time ever I'll be able to use my Windows PC and it will feel similar to i3 and/or Sway on Linux which is just FANTASTIC...

The code has been merged see https://github.com/RamonUnch/AltSnap/pull/564

I just built from source and it's perfect!!
